src/App.js

```diff
@@ -3,8 +3,18 @@ import Question from './Question.js';
 import questionData from './multipleQuestions.json';
 import logo from './logo.svg';
 import './App.css';
+import Counter from './Counter.js';
 
 class App extends Component {
+  getInitialState() {
+    return { count: 0 };
```

Comment:
This is absolutely the correct place to set the counter state, but maybe not the right way. We're using the ES6 class syntax so we can't use getInitialState. You'll need a constructor instead. Read this to see an example and understand more: https://facebook.github.io/react/docs/react-without-es6.html#setting-the-initial-state

You might also want to set a "currentQuestion" state here.